File: Assets/Tests/KotlinTokenizer/structs.kt

internal data class Data(internal var text: String) {}

data class Person(
    internal val name: String,
    internal val surname: String,
    internal var age: Int) {
    
    internal fun eat() {}

    private fun incrementAge() {}
}

internal data class User(
    var id: Int? = 0,
    var name: String? = null,
    var content: Content): Codable {
    
    data class Content(val text: String) {}
}

File: Assets/Tests/KotlinTokenizer/structs.swift
struct Data {
    var text: String
}

public struct Person: Equatable {
    let name: String
    let surname: String
    var age: Int

    internal func eat() {}
    private func incrementAge() {}
}

internal struct User: Codable {
    struct Content {
        let text: String
    }
    var id: Int? = 0
    var name: String?
    var content: Content
}
